Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetHeaderView.swift

```diff
@@ -17,7 +17,9 @@ class CreditCardBottomSheetHeaderView: UITableViewHeaderFooterView, ReusableCell
         static let mainContainerElementsSpacing: CGFloat = 7.0
         static let bottomSpacing: CGFloat = 24.0
         static let logoSize: CGFloat = 36.0
+        static let closeButtonMarginAndWidth = 46.0
```

Comment:
We should follow the same pattern as with the other constants.

```suggestion
        static let closeButtonMarginAndWidth: CGFloat = 46.0
```

Reply:
I've created [FXIOS-7038](https://mozilla-hub.atlassian.net/browse/FXIOS-7038) to address it, as it's not related to the current PR, and opened a [PR](https://github.com/mozilla-mobile/firefox-ios/pull/15650) for it separately.